### PR TITLE
PR: Change of the label in the dependencies dialog

### LINKED ISSUE #3522: Dependencies-Dialog not updating correctly when installing dependencies while spyder is running
## Description
This is a minor bug I noticed while installing dependencies with spyder running.
Spyder recognises newly installed dependencies, but will not insert the installed version number in the dependency info (or this bug report as can be seen by the additional information/dependencies below: pandas version number is displayed as None, but it is labeled as (OK). ) My guess would be that the version number is only requested at startup while the label (OK/NOK) as well as the color for each dependency is checked when opening the dialog?

**What steps will reproduce the problem?**

1. Open spyder with missing optional dependencies 
2. Check dependencies-info using Help->Dependencies... 
3. Install any missing dependency using pip (in the same environment)
4. Check dependencies-info again (close and reopen if still open)

**What is the expected output? What do you see instead?**
The expected output would be the version of the installed dependency. Instead we see that the dependency is not red anymore and is labeled as (OK) but not version number is displayed.

**Please provide any additional information below**
Restarting spyder fixes this problem, as the version number is correctly displayed then.

Uninstalling dependencies while spyder is running causes a similar (maybe even more serious?) bug, as in the fact that spyder does not seem to realize that the dependency is missing at all (still shows the old version number and labels it as (OK)). I just do not think this is so serious as people usually know when they uninstalled something.

## Version and main components

* Spyder Version: 3.0.0 
* Python Version: 2.7.12
* Qt Versions: 4.8.7, PyQt4 (API v2) 4.11.4 on Linux

## Dependencies
```
jedi >=0.8.1     :  0.9.0 (OK)
matplotlib >=1.0 :  1.5.3 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
numpy >=1.7      :  1.11.1 (OK)
pandas >=0.13.1  :  None (OK)
pep8 >=0.6       :  1.7.0 (OK)
psutil >=0.3     :  4.3.1 (OK)
pyflakes >=0.5.0 :  1.3.0 (OK)
pygments >=2.0   :  2.1.3 (OK)
pylint >=0.25    :  1.6.4 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
rope >=0.9.4     :  0.10.3 (OK)
sphinx >=0.6.6   :  1.4.6 (OK)
sympy >=0.7.3    :  None (NOK)

```

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #3522

### CODE REVIEW

**Comment by dalthviz** [comment]
@ccordoba12  this is ready for review :)

File: spyder/widgets/dependencies.py [hunk 1]
```diff
@@ -145,7 +145,9 @@ def __init__(self, parent):
                               "them.<br><br>"
                               "<b>Note</b>: You can safely use Spyder "
                               "without the following modules installed: "
-                              "<b>%s</b> and <b>%s</b>")
+                              "<b>%s</b> and <b>%s</b>. Also note that new "
```
Comment (ccordoba12) [inline spyder/widgets/dependencies.py]: Great! I like a lot the wording you decided to use. Just a couple of comments:

1. Add a `Please` at the beginning of the note:

        Please also note that new ....

2. Add this note in a new line by adding a `<br><br>` like this:

        "<b>%s</b> and <b>%s</b>.<br><br>"
        "Please also note that new ...."

That's all :-)

**Comment by ccordoba12** [comment]
Thanks @dalthviz!

**Comment by dalthviz** [comment]
You’re welcome @ccordoba12 👍

**Pull request merged**